sceptre/config/reader.py

```diff
@@ -11,22 +11,24 @@
 import datetime
 import fnmatch
 import logging
-from os import environ, path, walk
-from pkg_resources import iter_entry_points
-import yaml
+import shutil
+from os import environ, path, walk, getcwd, mkdir
 
 import jinja2
+import yaml
```

Comment:
Would you mind turning off some of your editor's reformatting automation?  It would really help reviewers to not have to look thru things that have not really changed.  Thank you for this much needed PR.  I'm looking forward to reviewing it.

Reply:
Sorry! I updated the commit to leave out all formatting changes so it should be a bit clearer now. Thanks for reviewing!